### ISSUE TITLE
Dataset JSON format?

### ISSUE THREAD

**Issue by f-izzat** [lemonyhw/ID_vs_Spot_face_verification#1]
What is the format of './Data/NJN_Random_prototype_train_300000.json' in DWI_AM-softmax.py? and how does one separate which one is the ID image and the other Spot image? 

Also, in `train_DWI_AM-softmax.py` why do we use a false positive as the target in the train loop
```
for batch_idx, (data1, data2, _, target) in enumerate(train_loader):
```
where `(data1, data2, _, target)` corresponds to `imgPath1, imgPath2, target_true, target_false = self.imgList[index]`
